.github/workflows/android-build.yml

```
@@ -0,0 +1,74 @@
+        java-version: '11'
+        
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y p7zip-full ant lib32z1-dev lib32stdc++6 python2.7
+        
+    - name: Start Xvfb
+      run: |
+        sudo apt-get install -y xvfb
+        Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)